### PR TITLE
Alerting/Docs: Silence documentation

### PR DESCRIPTION
**Pull request by kylebrandt**
Document silences for unified alerting.

### CODE REVIEW

File: docs/sources/alerting/unified-alerting/silences.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+description = "Silence alert notifications"
+keywords = ["grafana", "alerting", "silence", "mute"]
+weight = 400
++++
+
+# Silence alert notifications
+
+Grafana allows to you to prevent notifications from one or more alert rules by creating a silence. This silence lasts for a specified window of time.
+
+Silences do not prevent alert rules from being run. They also do not stop alert instances being shown in the user interface. Silences only prevent notifications from being created.
```
Comment (davidmparrott) [inline docs/sources/alerting/unified-alerting/silences.md:12]:
```suggestion
Silences do not prevent alert rules from being evaluated. They also do not stop alert instances being shown in the user interface. Silences only prevent notifications from being created.
```

**Review by davidmparrott — APPROVED** [review]

File: docs/sources/alerting/unified-alerting/silences.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## How label matching works
+
+Alert instances that have labels that match all of the "Matching Labels" specified in the silence will have their notifications suppressed.
+
+- The **Label** field is the name of the label to match. It must exactly match the label name.
+- The **Value** field matches against the corresponding value for the specified **Label** name. How it matches depends on the **Regex** and **Equal** checkboxes.
+- The **Regex** checkbox specifies if the inputted **Value** should be matched against labels as a regular expression. The regular expression is always anchored. If not selected it is an exact string match.
+- The **Equal** checkbox specifies if the match should include alert instances that match or do not match. If not checked, the silence will include alert instances _do not_ match.
+
```
Comment (davidmparrott) [inline docs/sources/alerting/unified-alerting/silences.md:33]: Do you think it's worth adapting this explanation from Domas? 

These fields translate to matcher operators as follows:  not regex and not equal is !=,  regex and not equal is !~, not regex and equal is =, regex and equal is =~

File: docs/sources/alerting/unified-alerting/silences.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
++++
+
+# Silence alert notifications
+
+Grafana allows to you to prevent notifications from one or more alert rules by creating a silence. This silence lasts for a specified window of time.
+
+Silences do not prevent alert rules from being run. They also do not stop alert instances being shown in the user interface. Silences only prevent notifications from being created.
+
+## Add a silence
+
```
Comment (achatterjee-grafana) [inline docs/sources/alerting/unified-alerting/silences.md:15]:
```suggestion

To add a silence:

```

File: docs/sources/alerting/unified-alerting/silences.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+1. In the Grafana menu hover your cursor over the **Alerting** (bell) icon and select **Silences** (crossed out bell icon).
```
Comment (achatterjee-grafana) [inline docs/sources/alerting/unified-alerting/silences.md:16]:
```suggestion
1. In the Grafana menu, hover your cursor over the **Alerting** (bell) icon and then select **Silences** (crossed out bell icon).
```

File: docs/sources/alerting/unified-alerting/silences.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+1. Click the **New Silence** button.
+1. Select the start and end date for the **Silence start and end** to indicate when the silence should go into effect and expire.
```
Comment (achatterjee-grafana) [inline docs/sources/alerting/unified-alerting/silences.md:18]:
```suggestion
1. Select the start and end date in **Silence start and end** to indicate when the silence should go into effect and expire.
```

File: docs/sources/alerting/unified-alerting/silences.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+1. Optionally, update the **Duration**, this will alter the time for the end of silence in the previous step to correspond to the start plus the duration.
```
Comment (achatterjee-grafana) [inline docs/sources/alerting/unified-alerting/silences.md:19]:
```suggestion
1. Optionally, update the **Duration** to alter the time for the end of silence in the previous step to correspond to the start plus the duration.
```

File: docs/sources/alerting/unified-alerting/silences.md [hunk 7]
```diff
@@ -0,0 +1,38 @@
...
+1. Click **Create**.
+
+## How label matching works
+
+Alert instances that have labels that match all of the "Matching Labels" specified in the silence will have their notifications suppressed.
+
+- The **Label** field is the name of the label to match. It must exactly match the label name.
+- The **Value** field matches against the corresponding value for the specified **Label** name. How it matches depends on the **Regex** and **Equal** checkboxes.
+- The **Regex** checkbox specifies if the inputted **Value** should be matched against labels as a regular expression. The regular expression is always anchored. If not selected it is an exact string match.
+- The **Equal** checkbox specifies if the match should include alert instances that match or do not match. If not checked, the silence will include alert instances _do not_ match.
```
Comment (achatterjee-grafana) [inline docs/sources/alerting/unified-alerting/silences.md:32]:
```suggestion
- The **Equal** checkbox specifies if the match should include alert instances that match or do not match. If not checked, the silence includes alert instances _do not_ match.
```

File: docs/sources/alerting/unified-alerting/silences.md [hunk 8]
```diff
@@ -0,0 +1,38 @@
...
+
+## Viewing and editing silences
+
+1. In the Grafana menu hover your cursor over the **Alerting** (bell) icon and select **Silences** (crossed out bell icon).
```
Comment (achatterjee-grafana) [inline docs/sources/alerting/unified-alerting/silences.md:36]:
```suggestion
1. In the Grafana menu hover your cursor over the **Alerting** (bell) icon, then select **Silences** (crossed out bell icon).
```

File: docs/sources/alerting/unified-alerting/silences.md [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+1. To end the silence, click the **Unsilence** button next to the listed silence.
```
Comment (achatterjee-grafana) [inline docs/sources/alerting/unified-alerting/silences.md:37]:
```suggestion
1. To end the silence, click the **Unsilence** option next to the listed silence.
```

File: docs/sources/alerting/unified-alerting/silences.md [hunk 10]
```diff
@@ -0,0 +1,38 @@
...
+1. To edit a silence, click the pencil icon next to the listed silence. This will take you to the same view as creating a silence, but with the form filed out. Here the fields can be changed and then press the **Submit** button to change it.
```
Comment (achatterjee-grafana) [inline docs/sources/alerting/unified-alerting/silences.md:38]:
```suggestion
1. To edit a silence, click the pencil icon next to the listed silence. Edit the silence using instructions on how to create a silence.
1. Click **Submit** to save your changes.
```

**Review by achatterjee-grafana — COMMENTED** [review]
Added some copy-edit suggestions.

**Review by achatterjee-grafana — APPROVED** [review]
LGTM

**Comment by domasx2** [comment]
IMHO worth mentioning that "unsilenced" silences stick around in the UI for up to 5 days and then are automatically cleaned up. There is no way to manually completely delete them and make them go away from the UI. This comes up sometimes

**Review by domasx2 — APPROVED** [review]

**Pull request merged**